systemtest/src/test/java/io/strimzi/systemtest/metrics/PrometheusST.java

```diff
@@ -0,0 +1,91 @@
+    }
+
+    @Test
+    public void testAlertManagerPodIsUp() {
+        assertThat("AlertManager pod not found", kubeClient().getPod(ALERTMANAGER_POD) != null);
+        int conditionsSize = kubeClient().getPod(ALERTMANAGER_POD).getStatus().getConditions().get(0).getStatus().length();
+        assertThat("AlertManager pod is not ready", kubeClient().getPod(ALERTMANAGER_POD).getStatus().getConditions().get(conditionsSize - 1).getStatus().equals("True"));
+    }
+
+    @Test
+    public void testPrometheusPodIsUp() {
+        assertThat("Prometheus pod not found", kubeClient().getPod(PROMETHEUS_POD) != null);
+        int conditionsSize = kubeClient().getPod(PROMETHEUS_POD).getStatus().getConditions().get(0).getStatus().length();
+        assertThat("Prometheus pod is not ready", kubeClient().getPod(PROMETHEUS_POD).getStatus().getConditions().get(conditionsSize - 1).getStatus().equals("True"));
+    }
+
+    @Test
+    public void testSecretsCreated() {
+        assertThat("additional-scrape-configs secret not found", kubeClient().getSecret("additional-scrape-configs") != null);
+        assertThat("additional-scrape-configs secret does not contain key prometheus-additional.yaml", kubeClient().getSecret("additional-scrape-configs").getData().get("prometheus-additional.yaml") != null);
+        assertThat("alertmanager-alertmanager secret", kubeClient().getSecret("alertmanager-alertmanager") != null);
+        assertThat("alertmanager-alertmanager secret does not contain key alertmanager.yaml", kubeClient().getSecret("alertmanager-alertmanager").getData().get("alertmanager.yaml") != null);
+    }
+
+    // No need to recreate environment after failed test. Only values from collected metrics are checked
+    @Override
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces) { }
+
+    @BeforeAll
+    void setupEnvironment() {
+        LOGGER.info("Creating resources before the test class");
+        prepareEnvForOperator(NAMESPACE);
+
+        createTestClassResources();
+        cmdKubeClient().apply(StUtils.downloadYamlAndReplaceNameSpace("https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml", NAMESPACE));
+
+        cmdKubeClient().exec("oc", "create", "secret", "generic", "additional-scrape-configs", "--from-file=prometheus-additional.yaml=../metrics/examples/prometheus/additional-properties/prometheus-additional.yaml", "-n", NAMESPACE);
+        cmdKubeClient().exec("oc", "create", "secret", "generic", "alertmanager-alertmanager", "--from-file=alertmanager.yaml=../metrics/examples/prometheus/alertmanager-config/alert-manager-config.yaml", "-n", NAMESPACE);
+
+        StUtils.waitForDeploymentReady("prometheus-operator", 1);
+
+        cmdKubeClient().apply(StUtils.updateNamespaceOfYamlFile("../metrics/examples/prometheus/install/strimzi-service-monitor.yaml", NAMESPACE));
```

Comment:
@Frawless Is this flexible anough for testing things like releases etc. where the file structure might be different?

Reply:
Currently, the structure is same AFAIK, this should cause no issue I think.